core/src/main/kotlin/fr/sncf/osrd/standalone_sim/StandaloneSimulation.kt

```diff
@@ -422,7 +422,7 @@ private fun <T> DistanceRangeMap<T>.toRangeMap(): RangeMap<Double, T> {
     val res = ImmutableRangeMap.builder<Double, T>()
     for (entry in this) {
         if (entry.value != null)
-            res.put(Range.closed(entry.lower.meters, entry.upper.meters), entry.value!!)
+            res.put(Range.closedOpen(entry.lower.meters, entry.upper.meters), entry.value!!)
```

Comment:
It does the job, but it makes me think of getting rid of this legacy `RangeMap` (at least in `getElectrificationMap()`).
But it's a bit more work, so depending on the emergency and availability, maybe just add a TODO (and an issue?) ?

Reply:
I'll add a TODO

Reply:
I created the issue here: https://github.com/OpenRailAssociation/osrd/issues/8857